### PR TITLE
Added Dutch translation of README

### PR DESCRIPTION
**Pull request by jasperro**
I hope it is good enough.

### CODE REVIEW

File: README.nl.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Lees de [FAQ](https://github.com/swaywm/sway/wiki). Word lid van het [IRC
+kanaal](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway op
+irc.freenode.net).
+
+Als je de ontwikkeling van sway wilt ondersteunen, draag dan bij aan [SirCmpwn's
+Patreon-pagina](https://patreon.com/sircmpwn).
+
+## Releasehandtekeningen
+
+Releases worden gesingeerd met [B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A)
```
Comment (faho) [inline README.nl.md]: Typo: gesigneerd?

Comment (ddevault) [reply README.nl.md]: Someone else suggests 'ondertekend'

Comment (faho) [reply README.nl.md]: Both sound alright to me, but note that my dutch is shaky (never learned to spell, never used a computer in dutch).

**Comment by jasperro** [comment]
I think that 'ondertekend' is better, it is closer to the meaning of 'signed' but 'gesingeerd' is fine too.

File: README.nl.md (hunk 1 above)
Comment (MartijnBraam) [reply README.nl.md]: ondertekend is correct dutch for this.

File: README.nl.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+en gepubliceerd [op GitHub](https://github.com/swaywm/sway/releases).
+
+## Installatie
+
+### Uit een pakket
```
Comment (MartijnBraam) [inline README.nl.md]: "Via een pakket" would be a better translation

File: README.nl.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Sway is beschikbaar in vele distributies. Probeer het "sway"-pakket te installeren met jouw pakketbeheerapplicatie. Als het niet beschikbaar is, bekijk dan [deze wikipagina](https://github.com/swaywm/sway/wiki/Unsupported-packages)
+voor informatie over installatie in jouw distributie.
+
+Als je geïnteresseerd bent in het maken van pakketten voor je distributie, stuur een bericht in het IRC-
+kanaal of stuur een e-mail naar sir@cmpwn.com voor advies.
+
+### Compilatie uit broncode
```
Comment (MartijnBraam) [inline README.nl.md]: "Compilatie vanuit broncode" would be a better translation

File: README.nl.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+_\* Compileerafhankelijkheden_
+
+Voer deze opdrachten uit:
+
+    meson build
+    ninja -C build
+    sudo ninja -C build install
+
+Op systemen zonder logind, moet je bij het binaire bestand de suid instellen:
```
Comment (MartijnBraam) [inline README.nl.md]: "moet je bij het binaire bestand het suid bit instellen"

File: README.nl.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+## Configuratie
+
+Als je al i3 gebruikt, kopieer dan je i3-configuratie naar `~/.config/sway/config` en
+het zal zonder verdere configuratie werken. Kopieer anders het voorbeeldconfiguratiebestand naar
+`~/.config/sway/config`. Dit is meestal het bestand: `/etc/sway/config`.
+Voer `man 5 sway` uit voor informatie over het configureren van sway.
+
+## Uitvoeren
+
+Voer `sway` vanaf een TTY uit. Sommige display-managers werken mogelijk, maar worden niet ondersteund door
```
Comment (MartijnBraam) [inline README.nl.md]: "werken mogelijk" > "kunnen werken"

**Comment by jasperro** [comment]
It looks like it is ready to be merged

**Comment by ddevault** [comment]
Thanks!

**Pull request merged**